bin/check-jenkins-health.rb

```diff
@@ -79,13 +79,24 @@ def run
     https ||= config[:https] ? 'https' : 'http'
     testurl = "#{https}://#{config[:server]}:#{config[:port]}#{config[:uri]}"
 
-    r = if config[:https] && config[:insecure]
-          RestClient::Resource.new(testurl, timeout: config[:timeout], verify_ssl: false).get
-        elsif config[:https]
-          RestClient::Resource.new(testurl, timeout: config[:timeout], verify_ssl: true).get
-        else
-          RestClient::Resource.new(testurl, timeout: config[:timeout]).get
-        end
+    rest_client = if config[:https] && config[:insecure]
+                    RestClient::Resource.new(testurl, timeout: config[:timeout], verify_ssl: false)
+                  elsif config[:https]
+                    RestClient::Resource.new(testurl, timeout: config[:timeout], verify_ssl: true)
+                  else
+                    RestClient::Resource.new(testurl, timeout: config[:timeout])
+                  end
+    begin
+      r = rest_client.get { |response| response }
```

Comment:
this code does not raise errors if a server respond 500

Reply:
we already rescue on `RestClient::ExceptionWithResponse` what does only looking at the response give us?